docs/user/agent/agent-services.md

```diff
@@ -0,0 +1,35 @@
+# Agent-based Installer Services
+
+## Automated Flow
+
+![Automated Flow](./agent_installer_services-automated_flow.png)
+
+In the automated flow, all information needed to configure a cluster is included in the agent ISO.
+When the agent ISO is booted, agent-tui and nmstate libraries are copied to the host from initrd. This copy 
+step is required because the agent-tui and nmstate libraries are too big to fit into the Ignition config.
+
+* agent-interactive-console - checks the host has connectivity to the release image
+* agent - takes host inventory and performs validations and reports results back to assisted-service
+* selinux - updates the SELinux policy to allow chronyd access to container files and UNIX datagram socket 
+* pre-network-manager-config - applies any static network configs before NetworkManager is started
```

Comment:
One nit - since the network configs can also be used to define interfaces with DHCP, maybe remove "static"